### ISSUE TITLE
Mark as Watched Question

### ISSUE THREAD

**Issue by Kyzoku** [NabokD/pockettube#49]
How often does the videos marked as watched get backed up?  I marked probably a hundred or more videos as watched and I misclicked away from the Deck and when I went back to the Deck page everything I had marked as watched were unwatched again.  I had even did a backup, but apparently that info isn't saved that way.

Edit 1:
I've found that a few videos that aren't being marked are videos that YouTube has put a warning due to suicide or self-harm.  This doesn't explain the many others that aren't being marked though.

Edit 2:
After writing that first edit I refreshed the page and now the deck is showing hundreds of videos previously marked as watched and being unwatched.  Might I recommend adding your own watched function like Video Deck for YouTube has.  For now, even though I paid a year subscription I just can't use this no matter how much I want to.

**Comment by NabokD** [comment]
PocketTube is not saved the data about watching on a backup file, it's saved on YouTube directly in your account using YouTube history. Maybe when you marked more hundred videos at once youtube decides it is like unusual activity and blocks requests, for this reason, the extension doesn't have the option to multiply select watched videos.
I can suggest taking a break for several seconds between marks.

**Comment by Kyzoku** [comment]
I marked a video as watched and then actually watched another video.  I repeated this step a number of times and a few hours later I refreshed the deck and most if not all those videos were back as being unwatched, even the ones I actually watched.  YouTube seems to have a horrible tracking of watch history as a google search shows people complaining about it for years now.  I asked a friend and they said that they have watched every video on a channel but YouTube keeps showing them videos they know they've watched as unwatched.

I had high hopes for PocketTube and the only other extension that has a deck is Video Deck for YouTube and it seems to be a bit of abandonware at this point.  Anyway good luck with the extension and maybe I'll check back in on it in hopes that you perhaps decide to save watch data locally.
